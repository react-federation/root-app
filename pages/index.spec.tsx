import setup from 'utils/test/setup'

import Index from '.'

describe('Index', () => {
  it('should match snapshot', async () => {
    const { asFragment } = setup(Index)

    expect(asFragment()).toMatchSnapshot()
  })
})
