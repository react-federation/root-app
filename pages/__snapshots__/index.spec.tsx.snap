// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index should match snapshot 1`] = `
<DocumentFragment>
  .emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.emotion-0>*:not(style)~*:not(style) {
  margin-top: 0.5rem;
  -webkit-margin-end: 0px;
  margin-inline-end: 0px;
  margin-bottom: 0px;
  -webkit-margin-start: 0px;
  margin-inline-start: 0px;
}

.emotion-1 {
  padding-bottom: 2px;
}

.emotion-2 {
  display: inline-block;
  white-space: nowrap;
  vertical-align: middle;
  margin-right: 2px;
}

.emotion-3 {
  display: inline-block;
  white-space: nowrap;
  vertical-align: middle;
}

.emotion-4 {
  background-color: gray.50;
  padding: 4px;
  border-radius: md;
}

<div
    class="chakra-stack emotion-0"
  >
    <div
      class="emotion-1"
    >
      <span
        class="chakra-badge emotion-2"
      >
        Getting started
      </span>
      <span
        class="chakra-badge emotion-3"
      >
        React ⚛️
      </span>
    </div>
    <div
      class="emotion-4"
    >
      <div
        class="chakra-stack emotion-0"
      >
        <h2
          class="chakra-heading emotion-6"
        >
          Welcome to your frontend!
        </h2>
        <p
          class="chakra-text emotion-6"
        >
          Here you'll find examples of how you can implement things the right way!
        </p>
        <div
          class="emotion-6"
        >
          <ul
            class="list-style"
          >
            <li>
              <p
                class="chakra-text emotion-6"
              >
                Styling
              </p>
              <ul
                class="list-style"
              >
                <a
                  href="vscode://file//components/html-wrapper/index.tsx"
                >
                  <li>
                    CSS-in-JS (We prefer to use this one!)
                  </li>
                </a>
                <a
                  href="vscode://file//components/welcome/welcome.module.scss"
                >
                  <li>
                    CSS modules
                  </li>
                </a>
                <a
                  href="vscode://file//styles/main.scss"
                >
                  <li>
                    Classic CSS/SCSS
                  </li>
                </a>
              </ul>
            </li>
            <li>
              How to write components
            </li>
            <li>
              Data fetching
            </li>
            <li>
              Global state
            </li>
            <li>
              Routing
            </li>
            <li>
              Testing
            </li>
          </ul>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
